hack/validate/golangci-lint.yml

```diff
@@ -104,11 +98,6 @@ issues:
     - text: "SA1019: httputil.ErrPersistEOF"
       linters:
         - staticcheck
-    # FIXME temporarily suppress these for false positives in tests (see https://github.com/dominikh/go-tools/issues/1022)
```

Comment:
Ah! For some reason I missed that they fixed https://github.com/dominikh/go-tools/issues/1022 👍